setup.py

```diff
@@ -1124,11 +1117,58 @@ def print_box(msg):
 
 
 def main():
-    if BUILD_LIBTORCH_WHL and BUILD_PYTHON_ONLY:
+    global BUILD_LIBTORCH_WHL
+    global BUILD_PYTHON_ONLY
+    global PACKAGE_NAME
+
+    BUILD_LIBTORCH_WHL = os.getenv("BUILD_LIBTORCH_WHL", "0") == "1"
+    BUILD_PYTHON_ONLY = os.getenv("BUILD_PYTHON_ONLY", "0") == "1"
+    BUILD_TWO_WHEELS = os.getenv("SPLIT_BUILD", "0") == "1"
+
+    if BUILD_LIBTORCH_WHL and BUILD_PYTHON_ONLY:  # noqa: F823
         raise RuntimeError(
             "Conflict: 'BUILD_LIBTORCH_WHL' and 'BUILD_PYTHON_ONLY' can't both be 1. Set one to 0 and rerun."
         )
 
+    if BUILD_TWO_WHEELS:
+        setup_cmd = sys.argv[1]
+
+        if (
+            setup_cmd == "build"
+            or setup_cmd == "build_ext"
+            or setup_cmd == "sdist"
+            or setup_cmd == "develop"
+        ):
+            raise RuntimeError(
+                "At the moment the SPLIT_BUILD option only supports the "
+                "clean and install commands. Please rerun setup.py using "
+                "one of those two commands."
+            )
+
+            final_package_name = PACKAGE_NAME
+            PACKAGE_NAME = "libtorch"
```

Comment:
What are the final 2 package names ? Where we will store them in ``https://download.pytorch.org/whl/nightly/`` - we want to make sure there do not come to conflict with current torch packages